docs/contract-interface/payable-methods.md

```diff
@@ -24,10 +24,10 @@ Example:
 impl Contract {
     #[payable]
     pub fn take_my_money(&mut self) {
-        env::log_str("Thanks!");
+        near_sdk::env::log_str("Thanks!");
     }
     pub fn do_not_take_my_money(&mut self) {
-        env::log_str("Thanks!");
+        near_sdk::env::log_str("Thanks!");
```

Comment:
I didn't mean to suggest that the prefix is added here, but it might be more clear if it is, since env isn't imported